code-challenges.test.js

```diff
@@ -65,5 +175,42 @@ const vowelTester2 = "academy"
 const vowelTester3 = "challenges"
 // Expected output: 2
 
-
-// b) Create the function that makes the test pass.
+describe ("firstVowel", () => {
+    it("takes in a string and logs the index of the first vowel", 
+    () => {
+      expect(firstVowel(vowelTester1)).toEqual(1)
+      expect(firstVowel(vowelTester2)).toEqual(0)
+    })
+  })
+
+//   ReferenceError: firstVowel is not defined
+
+
+// // b) Create the function that makes the test pass.
+
+
+// Pseudo Coding
+// first I am going to create a function called firtVowel
+// Then I am going to have it take in a string loop through it so it can idenitfy the first vowel of the value
+// I will have it return a new string with new values
+// I recived help with this question
+
+const firstVowel = (string) => {
+    for (i=0; i < string.length; i++) {
+      if (string[i] == 'a' || string[i] == 'e' || string[i] == 'i' || string[i] == 'o' || string[i] == 'u')
+      return string.indexOf(string[i])
+    }
+  }
```

Comment:
Looks great!  There are so many different ways to approach this problem and this is definitely a great one.